R/plot_timeseries.R

```diff
@@ -0,0 +1,59 @@
+#' Plot time series
+#' @description
+#' Plot time series of net events on active arm
```

Comment:
If we want to avoid active/placebo consistently we should change active to "second treatment option" or something.